app/javascript/src/components/Dashboard/Contacts/ContactTable.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from "react";
+
+import { MenuHorizontal } from "@bigbinary/neeto-icons";
+import { Table, Typography, Avatar, Dropdown } from "neetoui/v2";
+
+const ContactTable = ({ data }) => {
+  const [pageNumber, setPageNumber] = useState(1);
+  const [pageSize, setPageSize] = useState(9);
+
+  const COLUMNS = [
+    {
+      key: "nameAndRole",
+      title: "NAME & ROLE",
+      render: ({ name, role, image_url }) => (
+        <div className="flex items-center space-x-3">
+          <Avatar size="large" user={{ name: "TEST", imageUrl: image_url }} />
+          <div>
+            <Typography style="h5" className="leading-5 neeto-ui-text-gray-800">
+              {name}
+            </Typography>
+            <Typography className="text-xs leading-5 neeto-ui-text-gray-600">
+              {role}
+            </Typography>
+          </div>
+        </div>
+      ),
+    },
+    {
+      dataIndex: "email",
+      key: "email",
+      title: "EMAIL",
+      width: 250,
+      render: value => <Typography className="text-sm">{value}</Typography>,
+    },
+    {
+      dataIndex: "created_at",
+      key: "created_at",
+      title: "CREATED AT",
+      width: 200,
+      render: value => <Typography className="text-sm">{value}</Typography>,
+    },
+    {
+      dataIndex: "action",
+      key: "action",
+      width: 50,
+      render: () => (
+        <Dropdown icon={() => <MenuHorizontal />} buttonStyle="text">
```

Comment:
Do we need to pass in the icon with JSX? Couldn't you use just pass in the function?

Reply:
The styling is applied differently when I passed the function.

Reply:
@shamilsdq 
That's weird. Can you create a video showing what exactly are the changes that you are noticing in UI while using these two different patterns? Thanks.

Reply:
https://loom.com/share/02a4b1bb80044ae0b5568e7bee4d7206